frontend/packages/dev-console/src/components/import/pipeline/PipelineSection.tsx

```diff
@@ -0,0 +1,41 @@
+import * as React from 'react';
+import { connectToFlags, FlagsObject } from '@console/internal/reducers/features';
+import { getBadgeFromType } from '@console/shared';
+import { Split, SplitItem } from '@patternfly/react-core';
+import { useFormikContext, FormikValues } from 'formik';
+import { PipelineModel } from '../../../models';
+import { FLAG_OPENSHIFT_PIPELINE } from '../../../const';
+import { CheckboxField } from '../../formik-fields';
+import FormSection from '../section/FormSection';
+import PipelineTemplate from './PipelineTemplate';
+
+type PipelineSectionProps = {
+  flags: FlagsObject;
+};
+
+const PipelineSection: React.FC<PipelineSectionProps> = ({ flags }) => {
+  const { values } = useFormikContext<FormikValues>();
+
+  if (!values.image.selected) {
```

Comment:
may as well combine this and the following if condition into a single `return null`.

Reply:
Done.